src/compiler/config/test/validate-dev-server.spec.ts

```diff
@@ -276,4 +276,23 @@ describe('validateDevServer', () => {
     const { config } = validateConfig(inputConfig, mockLoadConfigInit());
     expect(config.devServer.prerenderConfig).toBe(wwwOutputTarget.prerenderConfig);
   });
+
+  describe('pingRoute', () => {
+    it('should default to /ping', () => {
+      const { config } = validateConfig(inputConfig, mockLoadConfigInit());
```

Comment:
Non-blocking - can we future proof this a little and make sure `pingRoute` isn't available on our config? 
```suggestion
      delete inputConfig.devServer.pingRoute;
      const { config } = validateConfig(inputConfig, mockLoadConfigInit());
```
I wouldn't be surprised if we added a default of '/ping' to the `inputDevServerConfig` in the future as a part of another effort and altered how this test is set up accidentally